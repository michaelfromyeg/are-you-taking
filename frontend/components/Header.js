import styles from "./Header.module.scss";

const Header = ({ text }) => {
  return <div className={styles.header}>{text}</div>;
};

export default Header;
