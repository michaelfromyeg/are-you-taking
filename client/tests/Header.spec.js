import renderer from "react-test-renderer";

import Header from "../components/Header";

test("Header", () => {
  const component = renderer.create(<Header text="Hello, world!" />);

  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
