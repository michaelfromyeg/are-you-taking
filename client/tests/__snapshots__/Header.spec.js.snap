// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header 1`] = `
<div>
  Hello, world!
</div>
`;
